src/main/resources/static/css/forms.css

```diff
@@ -21,10 +21,9 @@
 
 .footer {
     position: fixed;
-    left: 0;
+    z-index: 10;
+    height: 60px;
```

Comment:
when user is empty page than footer does on top that's why it was given fixed position

Reply:
Also noticed: why is the footer styling even in "forms.css"? Please move it.
Leave the position fixed, give it a margin and then give the "body" a margin-bottom that is about the height of the footer (so margin + size of text) so that this works. The margin does not have to be on the button

Reply:
footer is not part of my story and I haven't created it, so I don't know why its there. I had to change its styling cuz it wasn't working aligned with my subjects container

Reply:
you changed something in the footer - which you should not have, but you did, which git shows. So now please change it. It was part if the subjects because the button disappeared.

Reply:
I did